.travis.yml

```diff
@@ -158,33 +158,40 @@ install:
           if [ "${CACHED_DPDK_VERS}" != "${DPDK_VERS}" ]; then
             rm -rf dpdk
           fi
+        - if [ "${CC#clang}" != "${CC}" ] ;
+          then
+            DPDKCC=clang ;
```

Comment:
why do we need this and not just $CC ?

Reply:
It is used to:
 - set DPDK target directory (`...-clang` vs `...-gcc`)
 - set DPDK configuration items based on the toolchain selected